### PR TITLE
More run types

### PR DESCRIPTION
**Pull request by MaxNoe**
Jens wants to process more run types using erna.

For this we had to add several new columns to our database and update the filling scripts.

### CODE REVIEW

File: migration_scripts/update_existing_rows_drsstep.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from tqdm import tqdm
+import pandas as pd
+from erna.utils import load_config, create_mysql_engine
+import click
+
+
+@click.command()
+@click.option('-c', '--config')
+def main(config):
+
```
Comment (mackaiver) [inline migration_scripts/update_existing_rows_drsstep.py]: Add some docstrings please. What does this scipt do.

File: migration_scripts/add_runtype_columns.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import click
+
+from playhouse.migrate import MySQLMigrator, migrate
+from peewee import IntegerField, CharField
+
+
+@click.command()
+@click.option('--config', '-c', help='Path to the yaml config file')
+def main(config):
+    config = load_config(config)
```
Comment (mackaiver) [inline migration_scripts/add_runtype_columns.py]: Bad script. No docstring :P

File: migration_scripts/add_drs_step_and_roi.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from erna.utils import load_config
+import click
+
+from playhouse.migrate import MySQLMigrator, migrate
+from peewee import IntegerField, CharField
+
+
+@click.command()
+@click.option('--config', '-c', help='Path to the yaml config file')
+def main(config):
```
Comment (mackaiver) [inline migration_scripts/add_drs_step_and_roi.py]: When do I have to run any of these scripts? When I update the database?  Dont we have migrations for that?

Comment (MaxNoe) [inline migration_scripts/add_drs_step_and_roi.py]: This is how migrations are done in peewee. This is merely here to document what has been done.

File: migration_scripts/add_runtype_columns.py (hunk 2 above)
Comment (MaxNoe) [inline migration_scripts/add_runtype_columns.py]: This script has < 10 lines of code actually doing stuff

File: migration_scripts/add_drs_step_and_roi.py (hunk 3 above)
Comment (mackaiver) [inline migration_scripts/add_drs_step_and_roi.py]: I don't think this is a goodway to track schema changes over time. Maybe we can use something like this? https://github.com/klen/peewee_migrate

Comment (MaxNoe) [inline migration_scripts/add_drs_step_and_roi.py]: Looks promising.

**Pull request merged**